src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSelectionField.jsx

```diff
@@ -1,5 +1,237 @@
+                      disabled={loadingProviders}
+                    />
+                    <label htmlFor={`${idSchema.$id}_${provider.id}`}>
+                      <span className="vads-u-display--block vads-u-font-weight--bold">
+                        {name}
+                      </span>
+                      <span className="vads-u-display--block">
+                        {provider.address?.line}
+                      </span>
+                      <span className="vads-u-display--block">
+                        {provider.address.city}, {provider.address.state}{' '}
+                        {provider.address.postalCode}
+                      </span>
+                      <span className="vads-u-display--block vads-u-font-size--sm vads-u-font-weight--bold">
+                        {distance} miles
+                      </span>
+                    </label>
+                    {checked && (
+                      <button
+                        type="button"
+                        onClick={() => {
+                          onChange(
+                            communityCareProviderList.find(
+                              p => p.id === checkedProvider,
+                            ),
+                          );
+                          setCheckedProvider();
+                          setShowProvidersList(false);
+                        }}
+                      >
+                        Choose provider
+                      </button>
+                    )}
+                  </div>
+                );
+              })}
+            </>
+          )}
+        </>
+      )}
+      {requestStatus === FETCH_STATUS.succeeded &&
+        showProvidersList && (
+          <div className="vads-u-display--flex">
+            {requestStatus === FETCH_STATUS.succeeded &&
+              !loadingProviders &&
+              showProvidersList &&
+              providersListLength < communityCareProviderList.length && (
+                <>
+                  <button
+                    type="button"
+                    className="additional-info-button va-button-link vads-u-display--block vads-u-margin-right--2"
+                    onClick={() =>
+                      setProvidersListLength(providersListLength + 5)
+                    }
+                  >
+                    <span className="additional-info-title">
+                      +{' '}
+                      {Math.min(
+                        communityCareProviderList.length - providersListLength,
+                        INITIAL_PROVIDER_DISPLAY_COUNT,
+                      )}{' '}
+                      more providers
+                      <i className="fas fa-angle-down vads-u-padding-right--0p5" />
+                    </span>
+                  </button>
+                  <button
+                    type="button"
+                    className="vaos-appts__cancel-btn va-button-link vads-u-margin--0 vads-u-flex--0"
```

Comment:
We should also add an `aria-label` attribute here that's something like "Cancel choosing a provider", to give more context to screen readers.

Reply:
Good call, added.